app/controllers/application_controller/miq_request_methods.rb

```diff
@@ -168,7 +168,7 @@ def set_pre_prov_vars
 
     unless %w[image_miq_request_new miq_template_miq_request_new].include?(params[:pressed])
       path_to_report = ManageIQ::UI::Classic::Engine.root.join("product", "views", provisioning_report).to_s
```

Comment:
I'm assuming all of these are controlled by us?  That is, a user (via the UI/API) can't get their own files into this directory?

Reply:
Yup, those are ours `manageiq-ui-classic/product/views`, and `provisioning_report` will always return either `ProvisionCloudTemplates.yaml` or `ProvisionInfraTemplates.yaml`